tests/integration/test_coverage_gap_supplements.py

```
@@ -62,243 +62,156 @@ def get_feature_names_out(self) -> np.ndarray:
         return np.array(sorted(self.vocabulary_.keys()))
 
 
-def _make_embedder_with_fake_sklearn(tmp_path: Path | None = None, **kw: Any):
-    """Return a DocumentEmbedder instance backed by a fake sklearn."""
-    import file_organizer.services.deduplication.embedder as _mod
-
-    with (
-        patch.object(_mod, "_SKLEARN_AVAILABLE", True),
-        patch.object(_mod, "TfidfVectorizer", _FakeTfidfVectorizer),
+def _make_embedder_with_fake_sklearn(**kw: Any):
```

Ruff (0.15.9)

[warning] Missing return type annotation for private function `_make_embedder_with_fake_sklearn`

(ANN202)